includes/class-migration.php

```diff
@@ -164,6 +167,9 @@ public static function maybe_migrate() {
 		if ( \version_compare( $version_from_db, '4.7.1', '<' ) ) {
 			self::migrate_to_4_7_1();
 		}
+		if ( \version_compare( $version_from_db, '4.8.0', '<' ) ) {
```

Comment:
should we use `unreleased` as version, so that the release script has an easier job?

Reply:
🤯

Reply:
Fixed in 981b5c2.